walkthroughs/1-integrating-event-and-api-driven-apps/walkthrough.adoc

```diff
@@ -110,6 +113,46 @@ Verify that you followed each step in the procedure above.  If you are still hav
 
 // end::creating-api-connector[]
 
+
+// tag::creating-http-connection[]
+
+[id='creating-http-connection-in-fuse_{context}']
+[.integr8ly-docs-header]
+=== Creating HTTP connection to CRUD App
+
+To send messages to the Spring Boot CRUD app, you create a connection in Red Hat Fuse Online using the API connector you just created.
```

Comment:
"To send messages to the Spring Boot CRUD app" => "To enable Fuse Online to send messages from the queue to the Spring Boot..."